training_data_collector/main.py

```diff
@@ -0,0 +1,23 @@
+from operator import imod
+from selenium import webdriver
+from chessdotcom import actions
+from chessdotcom.labelGame import labelGame
+import time
+
+driver_location = "/snap/bin/chromium.chromedriver"
```

Comment:
If there's some setup involved with using this script can you include a README.md inside the subfolder on what needs to be installed/configured to use it?